lib/schema_generator/publisher_content_schema_generator.rb

```diff
@@ -23,7 +23,8 @@ def required
       required_properties = base_content_item["required"] + %w[document_type schema_name]
 
       # TODO: `contact` is a content item that is can be base path less.
```

Comment:
Fixed in 041637c. I'm not sure we really need that comment at all. I did consider deleting it.